### DISCUSSION TITLE
Using Azure Documentation on GitHub for Developing a RAG-Based Application - License and Copyrights

### DISCUSSION THREAD

**Discussion by iAMSagar44** [microsoft/RAG_Hack#35] [Q&A]
Hi,

I'm interested in using the Microsoft Azure documentation available on GitHub to develop a RAG based application (for e.g., these articles - [App Service](https://github.com/MicrosoftDocs/azure-docs/tree/main/articles/app-service). 
Specifically, I plan to load the markdown files and index them into a vector store. Do I need to obtain permission from Microsoft for this use, considering copyright and licensing issues?

Sagar

**Comment by artemious7** [comment]
I have the same question. @iAMSagar44 , would you like to join forces on the project?

**Comment by KhaoticMind** [comment]
Hi @iAMSagar44 , while not a direct answer to your question, on the root of the repo you provided there is a LICENSE file, containing the license the project (in the case all the docs) is licensed https://github.com/MicrosoftDocs/azure-docs/blob/main/LICENSE
On the right of the project if the info for this license (CC 4.0 in this case).

![image](https://github.com/user-attachments/assets/bae3f942-17a9-4623-b34b-c23a8261d154)
